src/main/java/com/epam/preprod/sirenko/FourElementsOfStringSumAsHashCodeWrapper.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.preprod.sirenko;
+
+/**
+ * String key wrapper, as HashCode uses sum of first four elements of string
+ *
+ * @author E.Sirenko
+ **/
+public class FourElementsOfStringSumAsHashCodeWrapper {
+		private String key;
+	
+	public FourElementsOfStringSumAsHashCodeWrapper() {
```

Comment:
We dont need constructor without param, because we need to set string field

Reply:
deleted